features/steps/102.ngsild.working_with_context.py

```diff
@@ -0,0 +1,18 @@
+from behave import given, step
+from config.settings import CODE_HOME
+from os.path import join
+from logging import getLogger
+import json
+
+__logger__ = getLogger(__name__)
+
+
+@given(u'I set the tutorial 102 NGSI-LD')
+def step_impl(context):
+    context.data_home = join(join(join(CODE_HOME, "features"), "data"), "102.ld.working_with_context")
+
+@step(u'I have the header "{header}" with value {value}')
```

Comment:
I prefer this definition:

I have the header "{header}" with value "{value}"

Reply:
Ok.

Reply:
LGTM